Sources/Shared/Extensions/WKWebViewExtensions.swift

```diff
@@ -90,4 +90,23 @@ public extension WKWebView {
       }
     }
   }
+  
+  @discardableResult @MainActor func evaluateSafeJavaScript(
+    functionName: String,
+    args: [Any] = [],
+    contentWorld: WKContentWorld,
+    escapeArgs: Bool = true,
+    asFunction: Bool = true
+  ) async -> (Any?, Error?) {
+    await withCheckedContinuation { continuation in
+      evaluateSafeJavaScript(
```

Comment:
This is probably a stupid question that I just glazed over and didn't figure out by myself, but what makes the JS safe in this case?

Reply:
This function is just a async wrapper around the existing function. The safe part of this function is the args are sanitized before being passed into the javascript in the [generateJSFunctionString function](https://github.com/brave/brave-ios/blob/solana-dapps/Sources/Shared/Extensions/WKWebViewExtensions.swift#L26). For this PR specific usage of this function, some [additional info](https://github.com/brave/brave-ios/pull/6050#discussion_r981247170) on why we weren't able to use the args param as often in this PR is in discussion above.

Reply:
Ok, gonna resolve this one then and we can consolidate the discussion into that comment above.